en/flight_controller/beaglebone_blue.md

```diff
@@ -65,102 +58,88 @@ Currently running on a:
 MODEL_BB_BLUE
 Robot Control library Version:
 1.0.0
-
 ```
 
-As of this writing, the latest OS images in which librobotcontrol works properly
-is [bone-debian-9.5-iot-armhf-2018-07-22-4gb.img.xz](https://rcn-ee.net/rootfs/bb.org/testing/2018-07-22/stretch-iot/bone-debian-9.5-iot-armhf-2018-07-22-4gb.img.xz).
+> **Tip** Optionally you can update to a realtime kernel, and if you do, re-check if *librobotcontrol* works properly with the realtime kernel.
 
-#### Setup Robot Control Library
+The latest OS images in which *librobotcontrol* works properly (at time of writing) is [bone-debian-9.5-iot-armhf-2018-07-22-4gb.img.xz](https://rcn-ee.net/rootfs/bb.org/testing/2018-07-22/stretch-iot/bone-debian-9.5-iot-armhf-2018-07-22-4gb.img.xz).
 
-If you want to build px4, there are additional setup steps for this library.
 
-It seems that currently librobotcontrol debian package is only available on 
-BeagleBoard products. Here are the ways to obtain the librobotcontrol on 
-BeagleBone Blue:
+### Setup Robot Control Library
 
-1. use the one pre-installed in BeagleBoard images
+If you want to build PX4, there are additional setup steps for this library.
 
-2. install from debian package or repository 
-```sh
-    sudo apt update && sudo apt install librobotcontrol
-```
+At time of writing the *librobotcontrol* debian package is only available on BeagleBoard products (not BeagleBone Blue). 
+Here are the ways to obtain the *librobotcontrol* on BeagleBone Blue:
```

Comment:
So just to confirm, you only need to do these on beaglebone blue?
Do you also need to get this library/do this if you are doing a cross-compiler build?

Reply:
BeagleBone Blue is part of BeagleBoard products, so librobotcontrol debian package is available on BeagleBone Blue. Currently, the build (make) system in librobotcontrol is setup in a way that does not allow cross compile (e.g., no configure script, C compiler is fixed to gcc, etc) so you cannot easily cross compile on other machine (e/g/, a machine with Intel CPU) without changing its build system. Therefore normal users can only get the library binary on BeagleBoard products (in our case, it's BeagleBone Blue). For cross compile dev computer, the user need copy the library binary and header files from BeagleBone Blue (our in rare case another BeagleBoard product, e.g., BeagleBone Black) to the dev computer.